projects/exui-common-lib/src/lib/gov-ui/components/gov-uk-date/gov-uk-date.component.scss

```diff
@@ -1,5 +1,5 @@
-#date-input-container {
-  margin-bottom: 30px;
+.date-input-container {
+  margin-bottom: 30px !important;
```

Comment:
@johnbenjamin-hmcts - hmm... is there another way of applying the style without using `!important`? It's usually frowned upon to use this override.